charts/spark-operator-chart/templates/deployment.yaml

```diff
@@ -101,6 +101,11 @@ spec:
         - -webhook-timeout={{ .Values.webhook.timeout }}
         - -webhook-namespace-selector={{ .Values.webhook.namespaceSelector }}
         - -webhook-object-selector={{ .Values.webhook.objectSelector }}
+          {{- if .Values.webhook.certManager.enable }}
```

Comment:
I don't think this is going to work, as these flags were removed in a previous [PR](https://github.com/kubeflow/spark-operator/pull/2016/files#diff-df64e37f2222b761c62bd8bf32c0347aae0fe1f46ff6acf0e66c2b61bd30d761L107-L112) by @ChenYi015 

However, there is value in being able to provide our own certs, perhaps they should be added back @ChenYi015

Reply:
@Cian911  Make sense, I will do it.

Reply:
@ChenYi015 I've accidentally discarded the changes in my fork -
but trying to re-apply them now looks a bit more complicated after the big controller-runtime restructuring.
What are your thoughts?